lib/Util.js

```diff
@@ -24,4 +31,14 @@ module.exports = class Util {
     throw new Error(`Unknown Algorithm: ${algorithm}`);
   }
 
+  static parseMacToDecArray(macAddress) {
+    const mac = [];
+    macAddress
+      .slice(0, 8)
+      .split(':') // TODO - is also a valid MAC address seperator
```

Comment:
I know this is not new but a regex like `const MAC_LIKE = /([0-9A-F]{2}[:-]){5}([0-9A-F]{2})/i;` would be better I think...